src/SwaggerProvider.DesignTime/v3/OperationCompiler.fs

```diff
@@ -384,18 +384,30 @@ type OperationCompiler (schema:OpenApiDocument, defCompiler:DefinitionCompiler,
             [
                 ProvidedConstructor(
                     [ProvidedParameter("httpClient", typeof<HttpClient>);
-                     ProvidedParameter("options", typeof<JsonSerializerOptions>, optionalValue = true)],
-                    invokeCode = (fun args ->
-                        match args with
-                        | [] -> failwith "Generated constructors should always pass the instance as the first argument!"
-                        | _ -> <@@ () @@>),
+                     ProvidedParameter("options", typeof<JsonSerializerOptions>)],
+                    invokeCode = (fun args -> <@@ () @@>),
                     BaseConstructorCall = fun args -> (baseCtor, args))
                 ProvidedConstructor(
-                    [ProvidedParameter("options", typeof<JsonSerializerOptions>, optionalValue = true)],
```

Comment:
Do optional parameters not work?

Reply:
`optionalValue` is not the the `bool` argument - [see here](https://github.com/fsprojects/FSharp.TypeProviders.SDK/blob/88696001fe4d5e841ef86955a6507bff3c27e4a6/src/ProvidedTypes.fsi#L24) 

SDK expect default value for provided parameter here, you code means `true :> JsonSerializerOptions`.
You may try to fix default parameters, but it may be tricky with `null` as default value.

Reply:
Ah, got it. I'll fix

Reply:
Try this
``` F#
[
    ProvidedConstructor(
        [ProvidedParameter("httpClient", typeof<HttpClient>);
            ProvidedParameter("options", typeof<JsonSerializerOptions>, optionalValue = (null:JsonSerializerOptions))],
        invokeCode = (fun args ->
            match args with
            | [] -> failwith "Generated constructors should always pass the instance as the first argument!"
            | _ -> <@@ () @@>),
        BaseConstructorCall = fun args -> (baseCtor, args))
    ProvidedConstructor(
        [ProvidedParameter("options", typeof<JsonSerializerOptions>, optionalValue = (null:JsonSerializerOptions))],
        invokeCode = (fun args -> <@@ () @@>),
        BaseConstructorCall = fun args ->
            let httpClient = <@ RuntimeHelpers.getDefaultHttpClient defaultHost @>
            let args' = args @ [httpClient]
            (baseCtor, args'))
] |> ty.AddMembers
```